keyboards/nafuda/readme.md

```diff
@@ -0,0 +1,17 @@
+# nafuda
+
+![nafuda](https://cdn-ak.f.st-hatena.com/images/fotolife/S/Salicylic_acid3/20190608/20190608024901.jpg)
+
+This is 7 keys cursor macropad. 
+
+Keyboard Maintainer: [Salicylic_acid3](https://github.com/Salicylic-acid3)  
```

Comment:
We updated this section in the template so it's a list instead of a paragraph. Lists don't need trailing whitespace in GitHub Flavored Markdown.

```suggestion
* Keyboard Maintainer: [Salicylic_acid3](https://github.com/Salicylic-acid3)
```